CDP.ts

```diff
@@ -12,6 +12,7 @@ export type DataCenter = 'eu5' | `us5` | `il1`;
 type StagingEnvs = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8;
 export type Env<n extends StagingEnvs = StagingEnvs> = 'prod' | `st${n}`;
 export const availableEnvs: Record<DataCenter, Env[]> = {
+    us5: [], // WIP: Rom B.
```

Comment:
* change to `us5: ['prod', ...createArray(1, n => `st${n + 1}` as Env)]`
* remove comment